_episodes/08-putting-it-all-together.md

```diff
@@ -146,25 +146,27 @@ styles and the source codes that create them.
 
 ### `plt` pyplot versus object-based matplotlib
 
-Matplotlib integrates nicely with the Numpy package and can use Numpy arrays
+Matplotlib integrates nicely with the numpy package and can use numpy arrays
 as input of the available plot functions. Consider the following example data,
-created with Numpy:
+created with numpy by drawing samples from a normal distribution:
 
 ~~~
 import numpy
-x = numpy.linspace(0, 5, 10)
-y = x ** 2
+mu, sigma = 0, 0.1
+s = np.random.normal(mu, sigma, 1000)
```

Comment:
no need for variables `mu` and `sigma` because they are not used anywhere

Reply:
```suggestion
s = np.random.normal(0, 0.1, 1000)
```

Reply:
Like I've suggested here?

Reply:
yes. I would suggest adding some extra info to the preceding sentence, something like:
>by drawing <this many> samples from a normal distribution with <such and such> mean value, <such and such> standard deviation

Reply:
Added this text in a new suggestion above.